Documentation for `cds add github-actions` (#2033)
cap-js/docs: guides/deployment/cicd.md

[[toc]]

## GitHub Actions

GitHub offers continuous integration using [GitHub Actions](https://docs.github.com/en/actions/automating-builds-and-tests/about-continuous-integration). In our [samples](https://github.com/capire/samples), we use simple workflows to [test, deploy and release new versions](https://github.com/capire/samples/tree/main/.github/workflows).

Add a default set of workflows to your project like so:

```sh
cds add github-actions
```
> You can also use `cds add gha` as a shortcut.

> ### Set Up a Sandbox

Author: Rephrased that entire section to use _Staging_. There's just a small tip left below to set up org-wide defaults for a sandbox setup.

> Deploying new projects to share them with project managers or testers can be simplified to allow for a seamless experience — even for entirely new projects — by setting up a shared sandbox environment.

Reviewer: Not sure why we need to stress new projects here.  I anyways expect them to work, and rather think existing projects are harder to fit.

```suggestion
Share deployments of projects with project managers or testers by setting up a shared staging environment.
```

Author: Rephrased that entire section.
